frontend/freebase2wikidata.js

```diff
@@ -34,9 +34,9 @@
 */
 
 $(function() {
-  'use strict';
```

Comment:
Do you have a good reason to drop the "use strict" option?

Reply:
We dropped it this [commit](https://github.com/Wikidata/primarysources/commit/d24ca9ecef71b93f0feed27f6fdc4e479c929004 ), if you remember, as a temporary fix for the tool. I also think that this is the source of the conflict as it is a duplicate commit.

Reply:
Ok, thank you! Sorry, for not remembering.